src/editor/Gui.cpp

```diff
@@ -3166,6 +3310,11 @@ void Gui::drawLightMapTool() {
 
 			if (showLightmapEditorUpdate)
 			{
+				logf("Clean previous lightmaps...\n");
+				for (int i = 0; i < MAXLIGHTMAPS; i++) {
+					sprintf(fileNam, "%s%s%s-%i.png", g_settings.gamedir.c_str(), g_settings.workingdir.c_str(), "lightmap", i);
+					removeFile(fileNam);
+				}

```

Comment:
This could erase someone's work, if they edited these lightmaps but clicked something else before importing them again. Safer to overwrite these if they exist, and only when the export button is pressed.

Reply:
Ok.